### PR TITLE
`hashsum`: add `--zero`  option (#4612)

### PR DESCRIPTION
**Pull request by 0xMRTT**
```
$ touch a
$ ./target/debug/coreutils md5sum --zero a
```

returns

```
d41d8cd98f00b204e9800998ecf8427e  a% 
```

With GNU:

```
$ /usr/bin/md5sum --zero  a
d41d8cd98f00b204e9800998ecf8427e  a% 
```

### CODE REVIEW

**Comment by 0xMRTT** [comment]
@tertsdiepraam Made a dumb error, now that's should be good...

**Comment by sylvestre** [comment]
Could you please add a test?

File: tests/by-util/test_hashsum.rs [hunk 1]
```diff
@@ -52,6 +52,14 @@ macro_rules! test_digest {
                 .no_stderr()
```
Comment (sylvestre) [inline tests/by-util/test_hashsum.rs:52]: Please run rustfmt on the test too

**Review by sylvestre — CHANGES_REQUESTED** [review]
needs rustfmt

**Pull request merged**